app/ds_config_sample.py

```diff
@@ -24,7 +24,8 @@
     "gateway_name": "stripe",
     "gateway_display_name": "Stripe",
     "github_example_url": "https://github.com/docusign/code-examples-python/tree/master/app/",
-    "documentation": ""  # Use an empty string to indicate no documentation path.
+    "documentation": "",  # Use an empty string to indicate no documentation path.
+    "quickstart": "true"
```

Comment:
this should not be set to true here, it comes from the quickstart code. It should be :
  "quickstart" : {QUICKSTART_VALUE}

Reply:
Oops! That was a mistake. Fixing!

Reply:
Fixed